File: 2014:2015/Maklerbuero/DartMaklerbuero/bin/classes/Objekt.dart
library Objekt;

class Objekt{
  int objNr;
  String agents_name;
  bool typus_buyus_rentus;
  double price;
  double area_size;
  
  /*constructor*/
  Objekt(int objnr, String an, bool tbr, double p, double as){
    objNr = objnr;
    agents_name = an;
    typus_buyus_rentus = tbr;
    price = p;
    area_size = as;
  }
  
  /*GET*/
  int getObjNr(){
    return objNr;
  }
  
  String getAgentsName(){
    return agents_name;
  }
  
  bool getTBR(){
    return typus_buyus_rentus;
  }
  
  double getPrice(){
    return price;
  }
  
  double getAreaSize(){
    return area_size;
  }
  /*GET*/
  
  /*SET*/
  void setObjrNr(int v){
    objNr = v;
  }
  
  void setAgentsName(String v){
    agents_name = v;
  }
  
  void setTBR(bool v){
    typus_buyus_rentus = v;
  }
  
  void setPrice(double v){
    price = v;
  }
  
  void setAreaSize(double v){
    area_size = v;
  }
  /*SET*/
}

File: 2014:2015/Maklerbuero/CppMaklerbuero/CppMaklerbuero/Classes/Objekt.cpp
//
//  Objekt.cpp
//  CppMaklerbuero
//
//  Created by Michael Köppl on 22/09/14.
//  Copyright (c) 2014 Michael Köppl. All rights reserved.
//

#include "Objekt.h"

Objekt::Objekt(int i, int o, std::string an, bool tbr, double p, double as){
    objId = i;
    objnr = o;
    agentsname = an;
    typus_buyus_rentus = tbr;
    price = p;
    area_size = as;
}

/*get*/
int Objekt::getId(){
    return objId;
}

int Objekt::getObjNr(){
    return objnr;
}

std::string Objekt::getAgent(){
    return agentsname;
}

bool Objekt::getTBR(){
    return typus_buyus_rentus;
}

double Objekt::getPrice(){
    return price;
}

double Objekt::getAreaSize(){
    return area_size;
}
/*get*/

/*set*/
void Objekt::setObjNr(int v){
    objnr = v;
}

void Objekt::setAgent(std::string v){
    agentsname = v;
}

void Objekt::setTBR(bool v){
    typus_buyus_rentus = v;
}

void Objekt::setPrice(double v){
    price = v;
}

void Objekt::setAreaSize(double v){
    area_size = v;
}
/*set*/